### PR TITLE
chore(plasma-infra): package lock v2

### PR DESCRIPTION
**Pull request by Yakutoc**
Плановый переход на **npm** версии **8.19.2**, идет в составе **Node v16.19.1**

`package-lock файлы` теперь 2-ой версии.

#### Алгоритм обновления: 

В корне проекта выполнил: 

`npm i --no-audit --lockfile-version 2 --package-lock-only --ignore-scripts`

Это команда обновила версию кореневого lockfile до второй, без установки самих зависимостей

В корне проекта выполнил:

`npx lerna exec -- npm i --no-audit --ignore-scripts --lockfile-version 2 --package-lock-only`

Это команда обновила версию lockfile для всех пакетов монорепозитория.  
    
В корне проекта выполнил: 

`npx lerna bootstrap`

#### Примечание: 

Проверял работоспособность 

- на чистом проекте
- с командой `npx lerna clean -y && rm -rf node_modules && npm ci --no-audit && npx lerna bootstrap -- --no-audit`

- Storybook запускается
- Тесты локальные в докере запускаются   

   
-----------

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.520.5111090014.swift)  
[color_sberHealth_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.520.5111090014.kt)  
[color_sberHealth_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.520.5111090014.ts)  
[color_sbermarket_business_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.520.5111090014.swift)  
[color_sbermarket_business_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.520.5111090014.kt)  
[color_sbermarket_business_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.520.5111090014.ts)  
[color_sbermarket_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.520.5111090014.swift)  
[color_sbermarket_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.520.5111090014.kt)  
[color_sbermarket_metro_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.520.5111090014.swift)  
[color_sbermarket_metro_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.520.5111090014.kt)  
[color_sbermarket_metro_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.520.5111090014.ts)  
[color_sbermarket_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.520.5111090014.ts)  
[color_sbermarket_selgros_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.520.5111090014.swift)  
[color_sbermarket_selgros_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.520.5111090014.kt)  
[color_sbermarket_selgros_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.520.5111090014.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.520.5111090014.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.520.5111090014.kt)  
[color_sbermarket_wlbusiness_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.520.5111090014.ts)  
[color_sberonline_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.520.5111090014.swift)  
[color_sberonline_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.520.5111090014.kt)  
[color_sberonline_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.520.5111090014.ts)  
[color_sberprime_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.520.5111090014.swift)  
[color_sberprime_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.520.5111090014.kt)  
[color_sberprime_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.520.5111090014.ts)  
[shadow_sbermarket_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.520.5111090014.ts)  
[typo_mage_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.520.5111090014.swift)  
[typo_mage_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.520.5111090014.kt)  
[typo_mage_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.520.5111090014.ts)  
[typo_plasma_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.520.5111090014.swift)  
[typo_plasma_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.520.5111090014.kt)  
[typo_plasma_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.520.5111090014.ts)  
[typo_ruler_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.520.5111090014.swift)  
[typo_ruler_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.520.5111090014.kt)  
[typo_ruler_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.520.5111090014.ts)  
[typo_sage_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.520.5111090014.swift)  
[typo_sage_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.520.5111090014.kt)  
[typo_sage_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.520.5111090014.ts)  
[typo_sbermarket_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.520.5111090014.swift)  
[typo_sbermarket_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.520.5111090014.kt)  
[typo_sbermarket_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.520.5111090014.ts)  
[typo_soulmate_ios-swift--canary.520.5111090014.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.520.5111090014.swift)  
[typo_soulmate_kotlin--canary.520.5111090014.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.520.5111090014.kt)  
[typo_soulmate_react-native--canary.520.5111090014.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.520.5111090014.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.215.1-canary.520.5111090014.0
  npm install @salutejs/plasma-temple@1.164.1-canary.520.5111090014.0
  npm install @salutejs/plasma-ui@1.196.1-canary.520.5111090014.0
  npm install @salutejs/plasma-web@1.215.1-canary.520.5111090014.0
  # or 
  yarn add @salutejs/plasma-b2c@1.215.1-canary.520.5111090014.0
  yarn add @salutejs/plasma-temple@1.164.1-canary.520.5111090014.0
  yarn add @salutejs/plasma-ui@1.196.1-canary.520.5111090014.0
  yarn add @salutejs/plasma-web@1.215.1-canary.520.5111090014.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

File: package.json [hunk 1]
```diff
@@ -84,7 +85,7 @@
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-cypress": "2.11.1",
     "eslint-plugin-flowtype": "4.7.0",
-    "eslint-plugin-import": "2.20.2",
+    "eslint-plugin-import": "2.22.1",
```
Comment (Yakutoc) [inline package.json:88]: [Note]: Нужно было что бы убрать ошибку - https://github.com/airbnb/javascript/issues/2331

https://github.com/airbnb/javascript/issues/2331#issuecomment-724114465

File: package.json [hunk 2]
```diff
@@ -64,6 +64,7 @@
     "@types/cypress-image-snapshot": "3.1.6",
     "@types/mocha": "9.0.0",
     "@types/node": "16.11.6",
+    "@types/react-native": "0.69.6",
```
Comment (Yakutoc) [inline package.json:67]: [Note]: Что бы обойти ошибки при сборке.

File: packages/plasma-b2c/package.json [hunk 3]
```diff
@@ -79,6 +75,7 @@
         "babel-plugin-annotate-pure-calls": "0.4.0",
         "babel-plugin-styled-components": "1.13.2",
         "babel-plugin-transform-class-properties": "6.24.1",
+        "default-browser-id": "1.0.4",
```
Comment (Yakutoc) [inline packages/plasma-b2c/package.json:78]: [Note]: 

Зависимость для корректной работы storybook.

Тоже самое для plasma-web/temple/ui

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: .github/workflows/performance-test.yml [hunk 4]
```diff
@@ -49,34 +49,36 @@ jobs:
...
           restore-keys: |
             npm-deps-${{ hashFiles('package-lock.json') }}
 
-      - name: Setup packages for master
+      - name: Setup packages for DEV
         run: |
-          npm install npm@6.14.17 -g
-          npm ci
-          npx lerna bootstrap
+          npm i -g --no-audit npm@6.14.17
```
Comment (Yakutoc) [inline .github/workflows/performance-test.yml:70]: [Note]: 

Оставил так как в dev сейчас лежат lock файлы старой версии. 

Из-за этого ошибки при сборке. 

В следующем ПР, где будет включаться кеш для npm, это строчка будет удалена. 

Потому что в dev уже будет лежать lock файлы 2 версии.

**Review by LamaEats — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

File: package.json (hunk 2 above)
Comment (neretin-trike) [reply package.json:67]: я всё ещё хотел бы понять, откуда у нас react-native на проекте :(

File: packages/plasma-b2c/package.json (hunk 3 above)
Comment (neretin-trike) [reply packages/plasma-b2c/package.json:78]: странная зависимость конечно, и [ответ ](https://github.com/storybookjs/storybook/issues/18268#issuecomment-1176266439)у них тоже странный. Интересно почему они сами не могли эту зависимость ставить по условию 🤔

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

File: package.json (hunk 2 above)
Comment (Yakutoc) [reply package.json:67]: @neretin-trike 

Ошибки была такого характера 

```
node_modules/@types/react-native/Libraries/StyleSheet/StyleSheetTypes.d.ts:15:5
```

С установкой этой версии, ошибка пропала. 

Screenshot 2023-05-29 at 12 52 11

Comment (Yakutoc) [reply package.json:67]: @neretin-trike Скорее всего как транзитивная зависимость 

Screenshot 2023-05-29 at 12 54 54

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

File: packages/plasma-b2c/package.json (hunk 3 above)
Comment (Yakutoc) [reply packages/plasma-b2c/package.json:78]: Хороший вопрос, можно завести issue у нас посмотреть что можно сделать что бы явно не ставить. Может с переходом на 7 версию storybook, отпадет необходимость.

File: package.json (hunk 2 above)
Comment (neretin-trike) [reply package.json:67]: ну вот да, кажется, что с `@types/styled-components` прилетает. Видимо если эту зависимость обновить до latest, то не придётся перебивать отдельной установкой `@types/react-native`

File: packages/plasma-b2c/package.json (hunk 3 above)
Comment (neretin-trike) [reply packages/plasma-b2c/package.json:78]: оке, заведи пожалуста

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: package.json (hunk 2 above)
Comment (Yakutoc) [reply package.json:67]: @neretin-trike В отдельной задаче давай попробую сделать, ок?

Так как зависимость это типы и разделе devDep.  

> Видимо если эту зависимость обновить до latest, 

есть сомнение

Screenshot 2023-05-29 at 13 39 46

Comment (neretin-trike) [reply package.json:67]: да, конечно 👌

File: packages/plasma-b2c/package.json (hunk 3 above)
Comment (Yakutoc) [reply packages/plasma-b2c/package.json:78]: https://github.com/salute-developers/plasma/issues/528

File: .github/workflows/cypress-ct-17.yml [hunk 5]
```diff
@@ -27,13 +27,14 @@ jobs:
...
     - name: Setup packages
       run: |
-        npm install npm@6.14.17 -g
-        npm i
-        npm i react@17 react-dom@17 --prefix="./packages/plasma-ui"
-        npm i react@17 react-dom@17 --prefix="./packages/plasma-temple"
-        npm i react@17 react-dom@17 --prefix="./packages/plasma-b2c"
-        npm i react@17 react-dom@17 --prefix="./packages/plasma-web"
-        npx lerna bootstrap
+        npm ci --no-progress --no-audit
```
Comment (Yeti-or) [inline .github/workflows/cypress-ct-17.yml:30]: не хотим `--no-audit` отдельным пр принести?
мы же сможем увидеть на графиках такие изменения?

Comment (Yeti-or) [reply .github/workflows/cypress-ct-17.yml:30]: `--no-progress` кстати на что тут влияет?

File: .github/workflows/documentation-main.yml [hunk 6]
```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           node-version-file: '.nvmrc'
 
-      # Данный шаг не исполняется, т.к. в настоящее время actions/cache поддерживается только для типов событий push и pull_request 
+      # Данный шаг не исполняется, т.к. в настоящее время actions/cache поддерживается только для типов событий push и pull_request
```
Comment (Yeti-or) [inline .github/workflows/documentation-main.yml:27]: и это все еще так?

Comment (Yeti-or) [reply .github/workflows/documentation-main.yml:27]: в новых версия cache всё еще нет поддержки?

File: .github/workflows/performance-test.yml (hunk 4 above)
Comment (Yeti-or) [reply .github/workflows/performance-test.yml:70]: TODO тогда сделай

File: package.json (hunk 2 above)
Comment (Yeti-or) [reply package.json:67]: только заведите плиз такую задачу

File: website/plasma-theme-builder/.npmrc [hunk 7]
```diff
@@ -0,0 +1,2 @@
+# disabled auto-installing peer dependencies
+legacy-peer-deps=true
```
Comment (Yeti-or) [inline website/plasma-theme-builder/.npmrc:2]: это потому что react-popper  тянет старый реакт?

**Review by Yeti-or — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

File: .github/workflows/performance-test.yml (hunk 4 above)
Comment (Yakutoc) [reply .github/workflows/performance-test.yml:70]: @Yeti-or В PR с включением cache этого уже нет.

File: .github/workflows/cypress-ct-17.yml (hunk 5 above)
Comment (Yakutoc) [reply .github/workflows/cypress-ct-17.yml:30]: @Yeti-or 

> Older versions of npm produced a progress bar as it installed packages, slowing down some systems (Windows) with the console interaction.

File: website/plasma-theme-builder/.npmrc (hunk 7 above)
Comment (Yakutoc) [reply website/plasma-theme-builder/.npmrc:2]: @Yeti-or Да, там начинает сыпать в консоль.

File: .github/workflows/documentation-main.yml (hunk 6 above)
Comment (Yakutoc) [reply .github/workflows/documentation-main.yml:27]: @Yeti-or В CI еще не пробовал. Я за этим прослежу

File: .github/workflows/cypress-ct-17.yml (hunk 5 above)
Comment (Yakutoc) [reply .github/workflows/cypress-ct-17.yml:30]: В принципе я особо не увидел каких то изменений. Но можно понаблюдать с неделю с этим, потом неделю без. ) И понять есть ли какое то влияние

File: website/plasma-theme-builder/.npmrc (hunk 7 above)
Comment (Yeti-or) [reply website/plasma-theme-builder/.npmrc:2]: Давай тут коммент:
# TODO: https://github.com/salute-developers/plasma/issues/529

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

File: .github/workflows/cypress-ct-17.yml (hunk 5 above)
Comment (Yakutoc) [reply .github/workflows/cypress-ct-17.yml:30]: @Yeti-or 

> не хотим --no-audit отдельным пр принести? 

А в чем логика? Объясни

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

File: .github/workflows/cypress-ct-17.yml (hunk 5 above)
Comment (Yakutoc) [reply .github/workflows/cypress-ct-17.yml:30]: > мы же сможем увидеть на графиках такие изменения?

Я надеюсь, что увидим.

Comment (Yakutoc) [reply .github/workflows/cypress-ct-17.yml:30]: @Yeti-or Статья 2019 года - http://www.tiernok.com/posts/2019/faster-npm-installs-during-ci/ 

Из нее и узнал про эти флаги.

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Review by kayman233 — APPROVED** [review]
Работоспособность проверил, все корректно прошло

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-520/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-520/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-520/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-520/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-520/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-520/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-520/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**